packages/babel-plugin-proposal-decorators/src/transformer-legacy.js

```diff
@@ -237,15 +237,18 @@ export default {
 
     const replacement = decoratedClassToExpression(decl);
     if (replacement) {
-      path.replaceWithMultiple([
-        replacement,
+      path.replaceWith(replacement);
+      path.insertAfter(
```

Comment:
The injected `_class` is a VariableDeclaration comes from `decoratedClassToExpression` when `decl.node.id` is empty. I think it suffices to register the VariableDeclaration instead of the ExportDeclaration.
```js
const [VariableDeclaration] = path.replaceWithMultiple(/* replacement */);
if (!decl.node.id) {
  path.scope.registerDeclaration(VariableDeclaration);
}
```

By the way `path.replaceWith` + `path.insertAfter` is more expensive than `path.replaceWithMultiple`.

Reply:
Thanks. Didn't realize `replaceWithMultiple` returned stuff.

Reply:
Done in 1a770d7